buildlib/tools/builds.sh

```diff
@@ -420,3 +445,8 @@ then
 	do_task 10 build_clang
 	do_task 10 build_armclang
 fi
+
+if [ "${build_static}" = "yes" ]
```

Comment:
i think can do static tests as part of long_test

Reply:
> i think can do static tests as part of long_test

it failed on most of rpm based dockers due to missing system static libraries installed, that is why it is tested on ubuntu-based dockers only